client/views/actions/topic.tpl

```diff
@@ -1,8 +1,7 @@
-{{#if from}}
+{{~#if from~}}
 	<span role="button" class="user {{colorClass from}}" data-name="{{from}}">{{mode}}{{from}}</span>
 	has changed the topic to:
-{{else}}
+{{~else~}}
 	The topic is:
-{{/if}}
-
+{{~/if~}}
 <span class="new-topic">{{{parse text}}}</span>
```

Comment:
This needs a space between the \: and the actual topic. Shows as `The topic is:welcome to this channel`